tests/testthat/test_MsExperiment-functions.R

```diff
@@ -97,6 +97,27 @@ test_that(".link_matrix works", {
     expect_equal(res, cbind(c(1, 2, 3, 5), c(2, 2, 3, 3)))
 })
 
+test_that(".link_matrix2 works", {
```

Comment:
Should be integrated into the original `link_matrix` tests.

Reply:
Done